### DISCUSSION TITLE
这个资源可以替换为国内的源吗？不翻墙加载不出来

### DISCUSSION THREAD

**Discussion by 124608760** [fvtt-cn/5etools#2] [General]
https://www.googletagmanager.com/gtag/js?id=UA-91432508-6  这个请求

**Comment by hmqgg** [comment]
这个是 G Tag 统计，用来大致跟踪、统计网站使用情况，并非必须加载出来，不影响网站正常使用。
并且，所有页面的 G Tag 也是在部署过程中，通过 CI/CD 流程自动加入的统计代码，不会影响自己从这个 Repo 同步下来部署的实例。

**Comment by hmqgg** [comment]
Google Analytics 在国内是合法运营，判断可能是没有过墙，但是被 Adblocker 之类的浏览器插件所拦截。
通常广告拦截器会拦截 Google Analytics 这种追踪统计分析脚本，如果需要的话，可以在 5etools 域名下关闭广告拦截器插件（或其他阻碍跟踪的浏览器功能/插件）。
